client/network/src/service.rs

```diff
@@ -1316,7 +1317,7 @@ where
 	/// that peer. Shared with the [`NetworkService`].
 	peers_notifications_sinks: Arc<Mutex<HashMap<(PeerId, ProtocolName), NotificationsSink>>>,
 	/// Controller for the handler of incoming and outgoing transactions.
-	tx_handler_controller: transactions::TransactionsHandlerController<H>,
+	tx_handler_controller: sc_network_transactions::TransactionsHandlerController<H>,
```

Comment:
This should be removed here as well. The idea is that the transaction protocol is not referenced anymore here anymore.

Reply:
Should it be moved to be part of `Behaviour`?